CONTRIBUTING.md

```diff
@@ -1,5 +1,20 @@
 # Contributing to Internet Identity
 
-We really appreciate bug reports through [GitHub tickets]. However **we currently do not accept pull requests**.
+Thank you for your interest in contributing to Internet Identity ❤️
```

Comment:
[ Internet Identity ]( https://identity.ic0.app/ ) ?

Reply:
Actually we have plenty of links already, if people land here they know what II is I think